libcaliptra/src/caliptra_api.c

```diff
@@ -0,0 +1,277 @@
+// Licensed under the Apache-2.0 license
+#include <stdbool.h>
+#include <stdint.h>
+#include <stdio.h>
+#include <string.h>
+#include <errno.h>
+#include <caliptra_top_reg.h>
+#include "caliptra_if.h"
+#include "caliptra_api.h"
+#include "caliptra_fuses.h"
+#include "caliptra_mbox.h"
+
+#define CALIPTRA_STATUS_NOT_READY 0
+
+static inline uint32_t caliptra_read_status(void)
+{
+    uint32_t status;
+
+    caliptra_read_u32(CALIPTRA_TOP_REG_GENERIC_AND_FUSE_REG_CPTRA_FLOW_STATUS, &status);
```

Comment:
Where is CALIPTRA_TOP_REG_GENERIC_AND_FUSE_REG_CPTRA_FLOW_STATUS defined?
I don't see it in this PR and I don't see it in the repo.

Reply:
`hw-latest/caliptra-rtl/src/soc_ifc/rtl/caliptra_top_reg.h
166:#define CALIPTRA_TOP_REG_GENERIC_AND_FUSE_REG_CPTRA_FLOW_STATUS                                     (0x3003003c)`